mergify_engine/github_types.py

```diff
@@ -86,6 +86,40 @@ class GitHubBranchCommit(typing.TypedDict):
     commit: GitHubBranchCommitCommit
 
 
+class GitHubTree(typing.TypedDict):
+    sha: SHAType
+
+
+class GitHubCommitVerification(typing.TypedDict):
+    verified: bool
+    reason: str
+    signature: str
+    payload: str
+
+
+class GitHubCommitAuthor(typing.TypedDict):
+    date: str
+    name: GitHubLogin
```

Comment:
I've a doubt but I don't this is a login

Reply:
yes, exact